From ba59b3eb4927f2e124c6cf5984d00da30f83fefe Mon Sep 17 00:00:00 2001
From: Slávek Banko <slavek.banko@axis.cz>
Date: Tue, 5 May 2026 00:09:10 +0200
Subject: libkonq: Define enum TextSortOrders as type name instead of a
 variable.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This resolves multiple definition of `TextSortOrders' with GCC 16.x.
This relates to issue #735.

Signed-off-by: Slávek Banko <slavek.banko@axis.cz>
---
 libkonq/konq_sort_constants.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

(limited to 'libkonq')

diff --git a/libkonq/konq_sort_constants.h b/libkonq/konq_sort_constants.h
index f6c695b19..91e1cf969 100644
--- a/libkonq/konq_sort_constants.h
+++ b/libkonq/konq_sort_constants.h
@@ -4,10 +4,10 @@
 typedef unsigned short TextSortOrder;
   // Can't use name 'SortOrder' because that's part of TQt
 
-enum {
+enum TextSortOrders {
   UNICODE_UNMODIFIED      = 0,
   LOCALE_UNMODIFIED       = 1,
   UNICODE_CASEINSENSITIVE = 2,
-} TextSortOrders;
+};
 
 #endif // KONQ_SORT_CONSTANTS_H
-- 
cgit v1.2.3

